internal/kokoro/environment.sh

```diff
@@ -29,6 +29,11 @@ if [[ -z "${PROJECT_ROOT:-}"  ]]; then
     PROJECT_ROOT="github/google-cloud-go"
 fi
 
+# add kokoro labels for testgrid filtering
+export PRODUCT_AREA_LABEL=observability
```

Comment:
Are these envvars already allow-listed in the internal kokoro config?

Reply:
yes